pkg/ndctl/region.go

```diff
@@ -188,8 +188,9 @@ func (r *Region) CreateNamespace(opts CreateNamespaceOpts) (*Namespace, error) {
 		ways := uint64(C.ndctl_region_get_interleave_ways(ndr))
 		align := opts.Align * ways
 		if opts.Size%align != 0 {
-			// force-align down to block boundary. More sensible would be to align up, but then it may fail because we ask more then there is left
+			// align up to block boundary.
 			opts.Size /= align
+			opts.Size += 1
```

Comment:
I think it would be cleaner to write this as:
```
// Round up size to align with next block boundary.
opts.Size = (opts.Size / align + 1) * align
```